### PR TITLE
Feature/pc 16

### PR DESCRIPTION
**Pull request by Cristofer16**
## summary
Login
##screenshots
image

### CODE REVIEW

**Comment by olmedo90** [comment]
Captura de pantalla (822)

es acorde al figma propuesto 👍

File: src/ui/components/button/button.jsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import { PropTypes } from "prop-types";
```
Comment (Ditmar) [inline src/ui/components/button/button.jsx:1]: use single quotes

File: src/ui/components/button/button.jsx [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 import { React } from "react";
```
Comment (Ditmar) [inline src/ui/components/button/button.jsx:2]: use single quotes

File: src/ui/components/loginT/login.css [hunk 3]
```diff
@@ -0,0 +1,147 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:ital,wght@0,700;0,900;1,900&display=swap');
+.main {
```
Comment (Ditmar) [inline src/ui/components/loginT/login.css:2]: don't mix the styles, if you want to use styles in a file do it, if you want to use the styles hardcoded do it, but use only one not both

File: src/ui/components/loginT/login.css [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    font-family: 'Mulish', sans-serif;
+    font-style: normal;
+    font-weight: bolder;
+    font-size: 0.75rem;
+    line-height: 0.938rem;
+    letter-spacing: 0.019rem;
+    text-transform: uppercase;
+    text-align: justify;
+}
+.dato {
```
Comment (Ditmar) [inline src/ui/components/loginT/login.css:98]: use english

File: src/ui/components/loginT/login.css [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+    align-items: center;
+    width: 18rem;
+    height: 2.625rem;
+    padding: 0.688rem 1rem;
+    box-sizing: border-box;
+    background: #FCFDFE;
+    border: 0.063rem solid #F0F1F7;
+    border-radius: 8px;
+}
+.boton {
```
Comment (Ditmar) [inline src/ui/components/loginT/login.css:109]: use english

File: src/ui/components/loginT/login.css [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+    padding: 1.5rem;
+}
+.ojoB {
```
Comment (Ditmar) [inline src/ui/components/loginT/login.css:112]: use english

File: src/ui/components/loginT/login.css [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+    position: absolute;
+    right: 3.6rem;
+    top: 23.6rem;
+    height: 18px;
+    width: 21px;
+}
+.ojo {
```
Comment (Ditmar) [inline src/ui/components/loginT/login.css:119]: use english

File: src/ui/components/loginT/login.css [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+    font-weight: 400;
+    font-size: 0.875rem;
+    line-height: 1.25rem;
+}
+.singup {
+    letter-spacing: 0.2px;
+    color: #3751FF;
+    font-weight: 600;
+}
+.olvidaste {
```
Comment (Ditmar) [inline src/ui/components/loginT/login.css:138]: use english

File: src/ui/components/loginT/login.jsx [hunk 9]
```diff
@@ -0,0 +1,44 @@
+import { Button } from "../button/button";
```
Comment (Ditmar) [inline src/ui/components/loginT/login.jsx:1]: use single quote

File: src/ui/components/loginT/login.jsx [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+import React from "react";
```
Comment (Ditmar) [inline src/ui/components/loginT/login.jsx:2]: use single quote

File: src/ui/components/loginT/login.stories.js [hunk 11]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import Login from "./login";
```
Comment (Ditmar) [inline src/ui/components/loginT/login.stories.js:2]: use single quotes

**Review by Ditmar — COMMENTED** [review]
needs works

**Review by Ditmar — APPROVED** [review]

**Pull request merged**